<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Patients</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-GestionPatients0-Desktop_Qt_5_13_0_MinGW_64_bit-Debug/ressources/Small_Logo_orange.png</normaloff>../build-GestionPatients0-Desktop_Qt_5_13_0_MinGW_64_bit-Debug/ressources/Small_Logo_orange.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonConnexionTcp">
        <property name="text">
         <string>Connexion Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDeconnexionTcp">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Déconnexion Box</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacer_16">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutAuthentification">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../build-GestionPatients0-Desktop_Qt_5_13_0_MinGW_64_bit-Debug/ressources/compte-utilisateur.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Authentification</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxUtilisateur">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Médecin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Infirmier</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Secrétaire</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Famille</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMotdepasse">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777206</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConnexionUtilisateur">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Connexion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDeconnexionUtilisateur">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Déconnexion</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_15">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../build-GestionPatients0-Desktop_Qt_5_13_0_MinGW_64_bit-Debug/ressources/user-search-icon.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Sélection d'un patient</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_20">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>N° de dossier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNumDossier">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMdpPatient">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonValidationPatient">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_18">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>ou</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAjoutPatient">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ajouter un patient</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_17">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutDonneesAdmin">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Données administratives</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>N° dossier</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditNumeroDossier">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maxLength">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_19">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditNom">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Prénom</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEditPrenom">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Adresse</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEditAdresse">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Code Postal</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEditCodePostal">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer_7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Ville</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="lineEditVille">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Age</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QSpinBox" name="spinBoxAge">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>122</number>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Sexe</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QComboBox" name="comboBoxSexe">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>------</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Femme</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Homme</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="15" column="1">
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="16" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Poids</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QSpinBox" name="spinBoxPoids">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>597</number>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <spacer name="verticalSpacer_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="18" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Groupe Sanguin</string>
           </property>
          </widget>
         </item>
         <item row="18" column="1">
          <widget class="QComboBox" name="comboBoxGroupeSanguin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>------</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>O-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>O+</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A+</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B+</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AB-</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AB+</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="19" column="1">
          <spacer name="verticalSpacer_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Date d'entrée</string>
           </property>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QLineEdit" name="dateEditDateEntree">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="21" column="1">
          <spacer name="verticalSpacer_13">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="22" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Lits</string>
           </property>
          </widget>
         </item>
         <item row="22" column="1">
          <widget class="QComboBox" name="comboBoxLit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>------</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1.a</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="24" column="1">
          <spacer name="verticalSpacer_14">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="25" column="1">
          <widget class="QPushButton" name="pushButtonEnregistrer">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Enregistrer</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../build-GestionPatients0-Desktop_Qt_5_13_0_MinGW_64_bit-Debug/ressources/secure_disquette.png</normaloff>../build-GestionPatients0-Desktop_Qt_5_13_0_MinGW_64_bit-Debug/ressources/secure_disquette.png</iconset>
           </property>
          </widget>
         </item>
         <item row="26" column="1">
          <widget class="QLabel" name="label_gif_chargement">
           <property name="text">
            <string>label_gif_chargement</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="23" column="1">
          <widget class="QPushButton" name="pushButtonLitsDispo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Lits disponibles</string>
           </property>
          </widget>
         </item>
         <item row="23" column="0">
          <widget class="QPushButton" name="pushButtonFinHospitalisation">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Fin d'hospitalisation</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutSanteChart">
      <item>
       <widget class="QPushButton" name="pushButtonChargementDonnees">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Charger les données</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
